README.md

```diff
@@ -1919,6 +1919,21 @@ To stop the playground from executing, simply call `PlaygroundPage.current.finis
 
 💦 Avoid memory leaks when accidentially refering to `self` in closures by overriding it locally with a weak reference:
 
+**Swift >= 4.2**
+
+```swift
+dataLoader.loadData(from: url) { [weak self] result in
+    guard let self = self else { return }
```

Comment:
Please follow the same code style as in the previous tip (that is, put the `return` on a new line) 😇